riscv/insns/vwsll_vi.h

```diff
@@ -0,0 +1,27 @@
+// vwsll.vi vd, vs2, zimm5, vm
+
+#include "zvk_ext_macros.h"
+
+// Uncomment to enable debug logging of invocations of this instruction.
+// #define DLOG_INVOCATION
+
+#if defined(DLOG_INVOCATION)
+#define DLOG(...) ZVK_DBG_LOG(__VA_ARGS__)
+// Print format/value for both state and round key element groups.
+#define PRIxEG PRIxEGU8x16
+#define PRVEG(X) PRVEGU8x16_LE(X)
+#else
+#define DLOG(...) (void)(0)
+#endif
+
+require_zvbb;
+
+DLOG("vwssl_vi vd#(%" PRIuREG "), vs2#(%" PRIuREG "), zimm5(%" PRIuREG ")"
+     " vstart(%" PRIuREG ") vl(%" PRIuREG ")",
+     insn.rd(), insn.rs2(), insn.v_zimm5(),
+     P.VU.vstart->read(), P.VU.vl->read());
+
```

Comment:
Delete.

Reply:
Done. I removed the debug logging throughout.

I understand that this logic was noisy and the desire to remove it. Please note however that developing this logging was critical to implement this functionality. The logs were modeled to closely match format used by each cryptographic algorithm specification (AES, SHA-2, SM3, SM4) in the respective sample sections. The logs found quite a few issues with the Zvk specification. As the Zvk specification is stable, its usefulness may indeed be over. Good luck to anyone trying to make significant changes to those instruction implementations in the absence of those logs.